src/pages/Content/index.js

```diff
@@ -1,3 +1,9 @@
+// import object selector to easily fetch DOM elements
+import { finder } from '@medv/finder'
+
+// Global recorded events array
+var recordedEvents = [];
```

Comment:
Make sure to merge the last PR before this one so we won't have conflicting histories.